src/pages/en/guides/content-collections.mdx

```diff
@@ -189,7 +189,7 @@ defineCollection({
     footnote: z.string().optional(),
     author: z.string().default('Anonymous'),
     language: z.enum(['en', 'es']),
-  }
+  })
 })
 ```
 
```

Comment:
We should probably also update the code sample in the “Advanced schema features” section below. (Thanks again for not allowing to comment there GitHub.)

Either wrap in `z.object` or maybe just remove the open/close braces?

```ts
// Allow only strings representing email addresses
authorContact: z.string().email(),
// Allow URL strings only (e.g. `https://example.com/`)
canonicalURL: z.string().url(),
// Parse publishDate as a browser-standard `Date` object
publishDate: z.string().transform(str => new Date(str)),
```

Reply:
Ah good catch! And yeah, GitHub reviews have been wacked-out lately. I'll fix